11/collection-meta.yaml

```
@@ -209,6 +209,14 @@ collections:
     repository: https://github.com/fortinet-ansible-dev/ansible-galaxy-fortios-collection
   google.cloud:
     repository: https://github.com/ansible-collections/google.cloud
+    removal:
+      major_version: 12
+      announce_version: 11.0.0a2
```

Assuming there is no further alpha release, this needs to be updated to:
```suggestion
      announce_version: 11.0.0b1
```